.travis.yml

```diff
@@ -24,11 +25,13 @@ matrix:
       env: SYMFONY_VERSION=3.4.*
     - php: 7.1
       env: SYMFONY_VERSION=4.0.*
+    - php: 7.2
+      env: SYMFONY_VERSION=4.0.*      
     - php: 5.6
       env: SYMFONY_VERSION=2.7.*;COMPOSER_FLAGS="--prefer-lowest"
 
 before_install:
-  - if [[ $TRAVIS_PHP_VERSION != '7.0' && $TRAVIS_PHP_VERSION != '7.1' ]]; then phpenv config-rm xdebug.ini; fi
+  - if [[ $TRAVIS_PHP_VERSION != '7.0' && $TRAVIS_PHP_VERSION != '7.1' && $TRAVIS_PHP_VERSION != '7.2' ]]; then phpenv config-rm xdebug.ini; fi
```

Comment:
i honestly wonder what is the purpose of this line @lsmith77

Reply:
disabling the xdebug extension

Reply:
but why for those versions

Reply:
mostly because php 7.0 is faster and more efficient so xdebug doesn't hurt so much .. IIRC